book/src/wasm_limitations.md

```diff
@@ -0,0 +1,26 @@
+# Limitations of WASM
+
+- Safari cannot spawn nested `Worker`s. To get around this limitation, build two variants of the library,
```

Comment:
nit: Would be great to let CI check this. (surely just a suggestion not a requirement!)

Reply:
I found [this repo](https://github.com/johanholmerin/nested-worker) while responding to your comment, might be interesting to check out.